### ISSUE TITLE
Repetitive MMS receiving 

### ISSUE THREAD

**Issue by Moostek** [moezbhatti/qksms#1950]
### DESCRIPTION
Hello, in last weeks I face the problem that I am receiving MMS from one sender every day (not in the same time). Now it happened to me also with SMS too, from different sender, but from same operator (T-Mobile). 

### STEPS
1. Repetitive, every day, MMS receiving. Now it is happenng also with one SMS. 

### EXPECTED
I expect I do not receive MMS/SMS repeatedly every day. 

### OBSERVATIONS
I expect I do not receive MMS/SMS repeatedly every day. 

I do not know if there is a problem with QKSMS, but the operator told me that there might be a problem with SMS/MMS app, i.e. QKSMS. 

Any idea? 
Thank you for your assistance.

**Comment by e-t-l** [comment]
Are these messages from people you know? If you don't want to receive them, you can block the sender by tapping the three-dot menu in the upper right corner and tapping Block.

This sounds like an issue with spam senders, not with QKSMS.

**Comment by Moostek** [comment]
Yes, you are probably right. I tried to resolve this with operators, but they were unable to find anything, they did recommend me to check the SMS app. Anyway, these repetitive SMS stopped after two weeks or so. So it is pointing on operator's issue.
